lib/std/c.zig

```diff
@@ -194,6 +194,20 @@ pub usingnamespace switch (builtin.os.tag) {
         pub extern "c" fn socket(domain: c_uint, sock_type: c_uint, protocol: c_uint) c_int;
         pub extern "c" fn stat(noalias path: [*:0]const u8, noalias buf: *Stat) c_int;
     },
+    .windows => struct {
+        // TODO: copied the else case and removed the socket function (because its in ws2_32)
+        //       need to verify which of these is actually supported on windows
```

Comment:
Can you reduce this list to only the functions that are needed by this pull request?

Reply:
none of these functions are needed, these function were already being defined for windows and I didn't want to change that functionality in this PR as it would be unrelated, but I can still remove them if you would like me to

Reply:
wait I think I didn't understand what happened here, let me take a second look.

Reply:
ohh! I see. Never mind what I said, what you did here makes sense.